packages/projects/src/components/ProjectForm.jsx

```diff
@@ -55,6 +55,9 @@ const INITIAL_STATE = {
   title: '',
   markdown: '',
   projects: null,
+  owner: null,
```

Comment:
I think it can be. I just intended it to be `null` originally and later ran into the error with the `select` input requiring an empty string, and fixed it at the input level instead of here, but I'll change it.